governance/blog-guidelines.md

```diff
@@ -0,0 +1,79 @@
+# Blog Guidelines
+
+## Introduction
+
+Welcome to the Security TAG (STAG) Blog! Our blog is a platform for sharing insights, experiences, and knowledge about Cloud Native software and the community. We encourage contributions from community members to help educate, inspire, and engage others. To ensure quality and consistency, we have established the following guidelines for content creation, submission review, and the overall value to the community.
+
+## Content Creation
+
+### Topics
+
+- **Cloud Native Projects**: Share updates, features, and success stories.
+- **Technical Tutorials**: Provide how-to guides, coding tips, and best practices.
+- **Community Stories**: Highlight contributors, events, and community achievements.
+- **Opinion Pieces**: Discuss trends, challenges, and the future of open source.
+
+### Writing Style
+
+- **Clarity**: Write in clear, concise language. Avoid jargon unless it's well-explained.
+- **Engagement**: Make your post interesting and engaging. Use anecdotes and examples.
+- **Accuracy**: Ensure all information is accurate and up-to-date.
+- **Inclusivity**: Use inclusive language and be respectful to all readers.
+
+### Formatting
+
+- **Headings**: Use appropriate headings (H1, H2, H3, etc.) to structure your post.
+- **Paragraphs**: Keep paragraphs short and focused.
+- **Lists**: Use bullet points or numbered lists to highlight key points.
+- **Tags**: Include relevant tags in the blog for filtering capabilities.
+- **Code**: Use code blocks for any code snippets.
+- **Images**: Include relevant images and graphics to enhance the content.
+
+### Length
+
+- Aim for 500-1200 words. Ensure the content is comprehensive yet concise.
+
+## Submission Review Process
+
+### Submission
+
+1. **Draft**: Write your blog post in a markdown file.
+2. **Submission Pull Request**: Submit your draft using the [blog submission template](https://github.com/cncf/tag-security/blob/main/.github/ISSUE_TEMPLATE/blog-submission.md).
```

Comment:
+1 . also IMHO we may need to clarify whether there is a naming convention for blog post markdown files and for the headings of blog posts so that they are displayed correctly in the website, e.g::

```markdown
---
title:  "Website Re-Launch at CNSC '24"
date:   2024-06-26 09:00:00 -0700
author: Eddie Knight 
---
```

Reply:
@mrcdb Added a section for `Front Matter` for article title. Appended an item to `Formatting` to include file naming conventions.

@eddie-knight Appended an item to Draft which contains the path to the blog article location.